test/health/rpc-health.test.ts

```diff
@@ -15,7 +15,24 @@ import { chainAddresses, chainMetadata } from '../../dist/index.js';
 import { Mailbox__factory } from '@hyperlane-xyz/core';
 
 const CHAINS_TO_SKIP = new Set([
-  'worldchain', // only allows private RPC access
+  // only allows private RPC access
+  'worldchain',
```

Comment:
i think we can use a public worldchain endpoint now?

Reply:
I'll try but i suspect we will get rate limited

Reply:
ok it worked fine it seems